package.json

```diff
@@ -4,7 +4,7 @@
 	"workspaces": [
 		"internal-packages/*",
 		"packages/*",
-		"pages-e2e/features/*"
+		"pages-e2e/*"
```

Comment:
why? 😮 

I wanted to add both `pages-e2e` but also keep apps inside `pages-e2e/features` as Greg set (so that on `npm i` we get everything I guess?)

without the `/*` we would only get the `pages-e2e` app, wouldn't we?

Reply:
it doesn't catch the workspace in `pages-e2e` at the moment since it's only looking at subdirectories, so turbo isnt running the command in any workspaces for `test:e2e`. you could put both `pages-e2e` and `pages-e2e/features/*` (from before) if you want both i guess

Reply:
ok it should be `/**` 😅 

but with that I get:
![Screenshot 2023-07-03 at 21 24 32](https://github.com/cloudflare/next-on-pages/assets/61631103/64c7c90f-0744-4b38-86ab-b3e0103fc52e)

so let's go with 
```
"pages-e2e",
"pages-e2e/features/*"
```

Thank a lot for noticing it! (stupid me for not testing it locally! 🤦)

Reply:
I have to look into this.... it looks like whatever dependencies I add to the fixtures don't get saved to the lock file nor are they present in the node_modules... 😕

Reply:
I think npm doesn't recognize workspaces inside a workspace 😓

Reply:
> I have to look into this.... it looks like whatever dependencies I add to the fixtures don't get saved to the lock file nor are they present in the node_modules... 😕

i think there may be a typo in one of the new additions that could be the cause of that

Reply:
ok that works, but it has the effect of not generating a lock file for the fixture

@GregBrimble is this acceptable for the pages-e2es? or do the app needs to have a lock file? (I think it does but maybe I'm wrong?)